1907-03-18.xml

```diff
@@ -0,0 +1,2174 @@
+                                </row>
+                            </table>
+                            <p>Ellerman <name>S.S. Bungarian</name> now loading for
+                                    <placeName>Liverpool</placeName> will sail for a few days to be
+                                followed by the <name>S.S. Alasatian</name>.</p>
+                            <p>N. E. TAMVACO Alexandria agents <measure type="indexNo"
+                                    >2202-90</measure></p>
+                        </div>
+                        <div type="advert" xml:id="deg-ad-ell03">
+                            <table rows="3" cols="6">
+                                <row>
+                                    <cell cols="3">Ellermans CITY LINE.</cell>
+                                    <cell cols="3">
+                                        <p>Ellermans CITY &amp; HALL LINES.</p>
+                                    </cell>
+                                </row>
+                                <row>
+                                    <cell cols="6">The undermentioned First Class Passenger Steamers
+                                        will be dispatched from Port Said on or about the following
+                                        dates for</cell>
+                                </row>
+                                <row>
+                                    <cell>Malta and London</cell>
+                                    <cell><name>S.S. City of Benares</name></cell>
+                                    <cell><date when="1907-03-19">March 19</date></cell>
+                                    <cell>Marseilles &amp; Liverpool</cell>
+                                    <cell><name>S.S. City of Vienna</name></cell>
+                                    <cell><date when="1907-04-04">April 4</date></cell>
+                                </row>
+                                <row>
+                                    <cell>Columbo &amp; Calcutta</cell>
+                                    <cell><name>S.S. City of Athens</name></cell>
+                                    <cell><date when="1907-03-19">March 19</date></cell>
+                                    <cell>Bombay &amp; Karachi</cell>
+                                    <cell><name>S.S. Trafford Hall</name></cell>
+                                    <cell><date when="1907-04-17">April 17</date></cell>
+                                </row>
+                            </table>
+                            <p>SALOON FARES:—Port Said to Malta £4.10.0. Marseilles. £10.0.0. London
+                                or Liverpool, £l4.l0.0. Colombo, Calcutta, Bombay or Karachi,
+                                £25.0.0. Special rates for steamers not carrying Doctor or
+                                Stewardess. For further particulars apply to</p>
+                            <p>CORY BROS. &amp; Co., Ltd., Agents for CITY Line, Port Sa
```

Comment:
1907

Reply:
?

Reply:
Sorry, I meant change the date at line 1115, which reads `<cell><date when="1908-04-17">April 17</date> ,,</cell>`